css/index.css

```diff
@@ -111,6 +139,115 @@ p {
     align-items: center;
     justify-content: center;
 }
+.box1 {
+    width: 12.5%;
+    height: 100px;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.box2 {
```

Comment:
Keep your code DRY. You need to create a single box and use it 10 time rather than creating it 10 times.